torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -810,7 +810,7 @@ def move_lifted_freevars_phs_to_end(
                     if mutation_info.has_mutation:
                         context = f"{mutation_info.msg} in\n {graph}"
                         unimplemented_v2(
-                            gb_type=f"Encountered input mutation during higher order op tracing for HOP - {source_target.name()}",
```

Comment:
Let's also move the dynamic parts to context/explanation (do the same for the other changes too).